### PR TITLE
Adds fonts section for SASS

### PR DESCRIPTION
**Pull request by tildedave**

### CODE REVIEW

**Comment by tildedave** [comment]
ping, also discuss

File: 02-sass.md [hunk 1]
```diff
@@ -1,7 +1,37 @@
...
+
+In general you should never need to override the font-family except for unusual circumstances such as styling an iframe.
+
+## More Information About Fonts
+
+We use font-weight and font-face together to avoid the browser choosing a "faux-bold".
+
+"Faux-bold", in addition to being ugly, will [break the display of colored emoji in Chrome](https://code.google.com/p/chromium/issues/detail?id=441946) (as of July 2015).
+
+We do not presently make usage of italics and there is no italic version of HelveticaNeueRoman currently available in the codebase.
```
Comment (marr) [inline 02-sass.md]: Should we add here, `We can still use `em` and `i` tags, but they will be faux italic.`

File: 02-sass.md [hunk 2]
```diff
@@ -1,7 +1,37 @@
...
+
+See also:
+
+* http://www.smashingmagazine.com/2013/02/setting-weights-and-styles-at-font-face-declaration/
+* http://alistapart.com/article/say-no-to-faux-bold
```
Comment (marr) [inline 02-sass.md]: ala++

File: 02-sass.md (hunk 1 above)
Comment (tildedave) [inline 02-sass.md]: yeah, good call

**Comment by tildedave** [comment]
Merging - let me know if there's more we should talk about!  (Maybe Quatro?)

**Pull request merged**